### ISSUE TITLE
Core dump on shutdown, uncaught "multipass::ExitlessSSHProcessException"

### ISSUE THREAD

**Issue by Saviq** [canonical/multipass#1127]
Multipass version `0.9.0-dev.298+g2a6fca71` dumps on shutdown:
```
paź 19 12:03:03 michal-laptop systemd[1]: Stopping Service for snap application multipass.multipassd...
paź 19 12:03:03 michal-laptop multipassd[9073]: Received signal 15 (Zakończony)
paź 19 12:03:03 michal-laptop multipassd[9073]: Goodbye!
paź 19 12:03:08 michal-laptop multipass.multipassd[9073]: terminate called after throwing an instance of 'multipass::ExitlessSSHProcessException'                                                                                            
paź 19 12:03:08 michal-laptop multipass.multipassd[9073]:   what():  failed to obtain exit status for remote process 'wall The system shutdown has been cancelled': timeout                                                                  
paź 19 12:03:08 michal-laptop systemd[1]: snap.multipass.multipassd.service: Main process exited, code=dumped, status=6/ABRT                                                                                                                 
paź 19 12:03:38 michal-laptop systemd[1]: snap.multipass.multipassd.service: State 'stop-final-sigterm' timed out. Killing.                                                                                                                  
paź 19 12:03:38 michal-laptop systemd[1]: snap.multipass.multipassd.service: Killing process 9186 (dnsmasq) with signal SIGKILL.                                                                                                             
paź 19 12:03:38 michal-laptop systemd[1]: snap.multipass.multipassd.service: Killing process 10831 (qemu-system-x86) with signal SIGKILL.                                                                                                    
paź 19 12:03:38 michal-laptop systemd[1]: snap.multipass.multipassd.service: Failed with result 'core-dump'.
```

**Comment by ricab** [comment]
I just tried and did not experience this:

```
out 21 10:23:22 ricab-xps13 systemd[1]: snap.multipass.multipassd.service: Succeeded.
out 21 10:23:22 ricab-xps13 systemd[1]: Stopped Service for snap application multipass.multipassd.
```

Any specific steps to reproduce?

**Comment by Saviq** [comment]
Not really, I'll come back if I find some.

**Comment by townsend2010** [comment]
I believe this has been fixed via #1172.  Closing.
